WeatherForecast/WeatherForecastTests/WeatherForecastTests.swift

```diff
@@ -8,24 +8,82 @@ import XCTest
 @testable import WeatherForecast
 
 class WeatherForecastTests: XCTestCase {
-
-    override func setUpWithError() throws {
-        // Put setup code here. This method is called before the invocation of each test method in the class.
+    
+    let parsingManager = ParsingManager()
+    
+    func test_JSON파일인CurrentWeather를_디코딩했을때_id는420006353이다() {
+        //given
+        let outputValue: Int?
+        let expectedValue = 420006353
+        let path = Bundle(for: type(of: self)).path(forResource: "CurrentWeather", ofType: "json")
+        let jsonFile = try! String(contentsOfFile: path!).data(using: .utf8)
+        //when
+        let data = parsingManager.parse(jsonFile!, to: CurrentWeather.self)
+        switch data {
+        case .success(let currentWeather):
+            outputValue = currentWeather.id
+        case .failure(_):
+            outputValue = -1
+        }
+        //then
+        XCTAssertEqual(expectedValue, outputValue)
     }
-
-    override func tearDownWithError() throws {
-        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    
+    func test_JSON파일인FiveDayWeather를_디코딩했을때_id는2643743이다() {
+        //given
+        let outputValue: Int?
```

Comment:
테스트코드 에러의 경우에는 `let` 으로 선언되어있어서 발생하는 에러입니다:) let 이기때문에 XCAssertEqual메서드가 불리기 전에 초기화되어야합니다. 따라서 위 에러가 발생하지않으려면 `var outputValue: Int?`로 변경해주거나, switch 문에서 failure 에서 nil을 할당해주세요 :)

```suggestion
        var outputValue: Int?
```

Reply:
아하.. let으로 선언해둔 걸 캐치하지 못했네요
가르쳐주셔서 감사합니다!